### ISSUE TITLE
Topaz Video Enhance AI crashes with an unknown error after momentary black screen

### ISSUE THREAD

**Issue by gargamel314** [IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT#270]
### Checklist [README]

- [X] Device is not undervolted nor overclocked
- [X] Device is using the [latest drivers](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/wiki/Intel-driver-updates-and-changelogs)
- [X] Application is not cracked, modded and use the latest patch

### Application [Required]

Topaz Video Enhance AI

### Processor / Processor Number [Required]

i7-13700K

### Graphic Card [Required]

Arc A770 LE 16GB

### GPU Driver Version [Required]

- [X] 31.0.101.4148
- [X] 31.0.101.4146
- [X] 31.0.101.4125
- [ ] 31.0.101.4123
- [ ] 31.0.101.4091
- [ ] 31.0.101.4090
- [ ] 31.0.101.4034
- [ ] 31.0.101.4032
- [ ] 31.0.101.3975
- [ ] 31.0.101.3959
- [ ] 31.0.101.3802
- [ ] 31.0.101.3793
- [ ] 31.0.101.3491
- [ ] 31.0.101.3490

### Rendering API [Required]

- [ ] Vulkan
- [ ] OpenGL
- [ ] DirectX12
- [ ] DirectX11
- [ ] DirectX10
- [ ] DirectX9
- [X] Not applicable

### Windows Build Number [Required]

- [X] Windows 11 22H2
- [ ] Windows 11 21H2
- [ ] Windows 10 22H2
- [ ] Windows 10 21H2
- [ ] Other (Please specify)

### Intel System Support Utility report

[SSU.txt](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/11009930/SSU.txt)

### Description and steps to reproduce [Required]

When you run Topaz Video Enhance AI, usually after around 5 or 10 minutes the screen will suddenly freeze, black out for about 3 seconds, and then come back. When everything comes back, VEAI shows the render failed with an "unknown error." I use two monitors: #1 is a Viewsonic 27" VX2758-2KP-MHD 2560x1440 @ 144Hz. Adaptive sync is enabled. The second is a Dell U2715H 2560x1440 @ 60Hz, it's not Adaptive-sync enabled. Both screens freeze and go black when this happens. Even after closing VEAI, I have noticed the screens black out on Windows Desktop. I've tried the Viewsonic at different refresh rates, even at 59Hz and 60 Hz. his never happens during any kind of gaming app, they remarkably run just fine. Thermals stay in the safe zones for all components and the power draw to the GPU doesn't go beyond 120W.

Things I've tried:
BIOS Settings - I had ASPM enabled w/ L1 enabled to control the idle power, I tried disabling and nothing changed.
Switching from DP cable to HDMI - no change
DDU - I've used it every time I've changed drivers.
Running at completely stock speeds - no overclock on the GPU or CPU. CPU is slightly undervolted for thermals.
Rolling back VEAI by completely uninstalling using REVO Uninstaller and reinstalling from scratch - same behavior on versions 3.1.6, 3.1.7, 3.1.8, 3.1.9.
Rolling back GPU driver. This had _some_ success. Everything is running fine on Arc Driver version 4125. This behavior black out screen comes with 4146 and now 4148, however I have seen the screen black out in 4125 when not running Topaz VEAI, but not nearly as frequently. Sometimes I can actually get it to complete a render before it goes. I did try to run a render today that was to last around 5 hours, but halfway through, I came back to it and saw it had failed with an unknown error. I've submitted a bug report with Topaz Tech Support as well.

### Device / Platform

ASUS Strix Z690-A D4

### Application / Windows logs

[logsForSupport.tar.gz](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/11009929/logsForSupport.tar.gz)

**Comment by Arturo-Intel** [comment]
I will work on this, 

Can you share the Topaz bug link?

**Comment by Arturo-Intel** [comment]
Oh, ok!
thank you for the info, rest assured that our development team is aware of this issue and they are working on it. Just remember that fixes can take 3-6 months to be implemented in a public divers version. 

Thank you for your feedback

**Comment by gargamel314** [comment]
Understood. Thank you.

**Comment by IGCIT** [comment]
hi @gargamel314,

please keep the issue open until a fix has landed

**Comment by gargamel314** [comment]
Hi there - FYI I made a video of this whole experience. It is here for your viewing pleasure: https://drive.google.com/file/d/1pAP6MvT-_an2jgodl_5PmwUFjpzoCZhc/view?usp=share_link

**Comment by BelleNottelling** [comment]
Hi there, I just wanted to share my experience here for anyone else having issues.
Topaz Video AI v3.2.0 included a fix for the A770 / A750. This release came out 11 days ago on the 4th, but even then I was having system and GPU driver crashes with my A770 and Topaz Video AI.

At least I was.. until I went into my BIOS and I turned back on the iGPU in my system. It now seems to be stable with the most recent driver (4311) and version of Topaz Video AI (3.2.1)
![image](https://user-images.githubusercontent.com/17304943/232277103-e7c33a72-e58e-47f8-81e0-e860fc196f3a.png)

**Comment by Arturo-Intel** [comment]
Hi @gargamel314 

Just want to confirm if you tried with v4311

--r2

**Comment by gargamel314** [comment]
YES i did. Also with 4335. Same results.

**Comment by Arturo-Intel** [comment]
OK, thanks for the confirmation

-- r2

**Comment by BelleNottelling** [comment]
I just did some more testing with my new PC & the recently released 4382 drivers.
The TLDR: It still does not work in any situation I have tested, although various configurations seems to improve the stability a bit, but not enough to make the software at all usable.

Each time I tested a render, it was with the same video and the same 2x preset (the default one, which selects the Proteus mode).
It was done with the `ProRes 422 LT` encoder as previously Topaz has suggested to avoid using the intel encoders when testing on Arc. Although, based on my previous experience the only situation where this mattered was with the AV1 encoder as it often had artifacts in the encoded video.

# Tested configurations
## Video Enhance AI with Re-bar and no iGPU:
Benchmark results:
```
Topaz Video AI  v3.2.8
System Information
OS: Windows v11.2009
CPU: 13th Gen Intel(R) Core(TM) i7-13700K  63.768 GB
GPU: Intel(R) Arc(TM) A770 Graphics  15.875 GB
Processing Settings
device: 0 vram: 1 instances: 0
Input Resolution: 1920x1080
Benchmark Results
Artemis		1X: 	ERR fps 	2X: 	ERR fps 	4X: 	01.83 fps 	
Proteus		1X: 	ERR fps 	2X: 	ERR fps 	4X: 	01.57 fps 	
Gaia		1X: 	05.52 fps 	2X: 	03.65 fps 	4X: 	02.81 fps 	
4X Slowmo		Apollo: 	07.56 fps 	APFast: 	21.68 fps 	Chronos: 	05.67 fps 	CHFast: 	09.59 fps 	
```
I skipped attempting to render anything with this test, as the Proteus 2x benchmark failed so there was clearly no point.

## Video Enhance AI with Re-bar and the iGPU:
Benchmark results:
```
Topaz Video AI  v3.2.8
System Information
OS: Windows v11.2009
CPU: 13th Gen Intel(R) Core(TM) i7-13700K  63.768 GB
GPU: Intel(R) Arc(TM) A770 Graphics  15.875 GB
GPU: Intel(R) UHD Graphics 770  0.125 GB
Processing Settings
device: 0 vram: 1 instances: 0
Input Resolution: 1920x1080
Benchmark Results
Artemis		1X: 	11.02 fps 	2X: 	05.84 fps 	4X: 	01.85 fps 	
Proteus		1X: 	09.52 fps 	2X: 	04.67 fps 	4X: 	01.59 fps 	
Gaia		1X: 	05.56 fps 	2X: 	03.64 fps 	4X: 	02.83 fps 	
4X Slowmo		Apollo: 	07.23 fps 	APFast: 	20.44 fps 	Chronos: 	05.62 fps 	CHFast: 	09.54 fps 	
```
In this case, the render failed within roughly 5 minutes.

## Video Enhance AI without Re-bar and no iGPU:
Benchmark results:
```
Topaz Video AI  v3.2.8
System Information
OS: Windows v11.2009
CPU: 13th Gen Intel(R) Core(TM) i7-13700K  63.768 GB
GPU: Intel(R) Arc(TM) A770 Graphics  15.859 GB
Processing Settings
device: 0 vram: 1 instances: 0
Input Resolution: 1920x1080
Benchmark Results
Artemis		1X: 	09.72 fps 	2X: 	05.87 fps 	4X: 	01.88 fps 	
Proteus		1X: 	09.00 fps 	2X: 	04.69 fps 	4X: 	01.17 fps 	
Gaia		1X: 	05.46 fps 	2X: 	03.59 fps 	4X: 	02.41 fps 	
4X Slowmo		Apollo: 	07.53 fps 	APFast: 	20.46 fps 	Chronos: 	05.61 fps 	CHFast: 	08.93 fps 	
```
This time, the render failed 5-10 minutes. It ran the longest than the other configurations I tested, but it seems fairly random when it does crash so I'm not sure how significant this actually is.

## Video Enhance AI without Re-bar and the iGPU:
Benchmark results:
```
Topaz Video AI  v3.2.8
System Information
OS: Windows v11.2009
CPU: 13th Gen Intel(R) Core(TM) i7-13700K  63.768 GB
GPU: Intel(R) Arc(TM) A770 Graphics  15.859 GB
GPU: Intel(R) UHD Graphics 770  0.125 GB
Processing Settings
device: 0 vram: 1 instances: 0
Input Resolution: 1920x1080
Benchmark Results
Artemis		1X: 	10.50 fps 	2X: 	05.91 fps 	4X: 	01.87 fps 	
Proteus		1X: 	09.50 fps 	2X: 	04.76 fps 	4X: 	01.53 fps 	
Gaia		1X: 	05.22 fps 	2X: 	03.56 fps 	4X: 	02.80 fps 	
4X Slowmo		Apollo: 	07.27 fps 	APFast: 	20.55 fps 	Chronos: 	05.46 fps 	CHFast: 	09.16 fps 	
```
The render failed within about 5 minutes with this configuration.

**Comment by gargamel314** [comment]
![image](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/124757309/1537b8c0-5e22-48ad-b206-567eae548e64)

Hey Intel. It's been 3 months. Are you going to fix this? I'm sorry, I know you said 3-6 months, but at least kindly give us an update on whether this is being addressed or swept under the rug. Thank you.

**Comment by BelleNottelling** [comment]
I personally would be happy to get occasional updates on what's going on with this issue.

Outside of how frequently DX11 is just terrible on these cards, the Gigapixel AI and Video Enhance AI issues are in my opinion the most disappointing. Gigapixel AI was shipped with the A770 in a bundle and is broken and even excluding that.. 
Topaz is literally listed as a partner on the [Deep Link Technology](https://www.intel.com/content/www/us/en/architecture-and-technology/adaptix/deep-link.html) page so it's a little surprising to see software coming from an Intel partner to be broken on Intel products.

**Comment by Karen-Intel** [comment]
Hey all. As you know this forum is actively being monitored.

Like Arturo mentioned **just last week**, the issue is being actively worked on.
We're providing status updates regularly and adding the impact you are making us aware of (users affected, etc.)

Also like I have mentioned in other threads, partnership means shared responsibility (50 Software vendor, 50 GFX HW provider) There are a lot of things going on behind the curtains and ongoing work on this issue + others we have submitted internally.
 
We appreciate your patience as we will be the ones updating the threads once a public fix has been released, asking the issue submitter to verify before even closing an issue. 

Karen

**Comment by Karen-Intel** [comment]
Update: No target date for fix to be released yet. However, we're still monitoring the thread internally :)

Karen

**Comment by BelleNottelling** [comment]
Hi, @Karen-Intel
Does that mean a fix has been found and it's undergoing testing before it can have a target release date?

**Comment by Karen-Intel** [comment]
> Hi, @Karen-Intel Does that mean a fix has been found and it's undergoing testing before it can have a target release date?

It means the issue has been confirmed and our dev team is still working on it. 
Just wanted to give you a heads up that the topic is still under our radar :)

K

**Comment by BelleNottelling** [comment]
> It means the issue has been confirmed and our dev team is still working on it.
> Just wanted to give you a heads up that the topic is still under our radar :)

Okay, thank you for the clarification

**Comment by gargamel314** [comment]
> Update: No target date for fix to be released yet. However, we're still monitoring the thread internally :)
> 
> Karen

Thank you for the update :) It's most appreciated! You people are awesome.

**Comment by Denizeri24** [comment]
Any update about this issue?

> > Hi, @Karen-Intel Does that mean a fix has been found and it's undergoing testing before it can have a target release date?
> 
> It means the issue has been confirmed and our dev team is still working on it. Just wanted to give you a heads up that the topic is still under our radar :)
> 
> K

**Comment by Arturo-Intel** [comment]
@Denizeri24 work in progress.

The team is actively working on this case. We will update this thread when we have any news.

Thank you for your patience.
-- r2

**Comment by Kikuzyu** [comment]
Hello. I have been having the same problem for quite a while until I suddenly found somewhat of a fix: by enabling my iGPU in the bios (which I'd disabled before) and installing the latest iGPU drivers while keeping my A770, I am now able to export my videos.
Now, I am still not sure if this fixes the problem entirely because my A770 is reported to be consuming 80W average on HWMonitor at a 69.5% peak GPU utilization, and I'm not sure that's the ideal behavior (it's the ASRock OC'd custom, I didn't underclock it or anything), but for now it's definitely better than me having to resort to a 750 Ti from Nvidia for exports lol. I jumped from 1 fps to 11 fps, which is definitely a good improvement.

GPUs are:
10600k (UHD 630)
A770 8GB (the only one driving my monitors, 10600k is connected to none)

Looking forward to a definitive fix to the problem! I hope the low card utilization will be addressed too x)

**Comment by Kikuzyu** [comment]
> Hello. I have been having the same problem for quite a while until I suddenly found somewhat of a fix: by enabling my iGPU in the bios (which I'd disabled before) and installing the latest iGPU drivers while keeping my A770, I am now able to export my videos. Now, I am still not sure if this fixes the problem entirely because my A770 is reported to be consuming 80W average on HWMonitor at a 69.5% peak GPU utilization (of course it doesn't always work at said power, it has sudden spikes, slows down momentarily, then goes up again- this loops until the video finishes export), and I'm not sure that's the ideal behavior (it's the ASRock OC'd custom, I didn't underclock it or anything), but for now it's definitely better than me having to resort to a 750 Ti from Nvidia for exports lol. I jumped from 1 fps to 11 fps, which is definitely a good improvement.
> 
> GPUs are: 10600k (UHD 630) A770 8GB (the only one driving my monitors, 10600k is connected to none)
> 
> Looking forward to a definitive fix to the problem! I hope the low card utilization will be addressed too x)
> 
> P.S.: Mind that this applies in my specific case and might not apply in your case. I have used a model that may differ from the one that you might wanna use, and a different one might break things as stated in Intel's drivers release notes, so keep that in mind x)

Update: It's actually a hit or miss fix, sometimes it finishes exporting the vids, sometimes it fails close to the last few %s :/

**Comment by Kikuzyu** [comment]
Hello everyone, Topaz 3.4.0 came out recently and the release notes mention a potential fix to Intel Arc processing issues.
I suggest everyone who's had the issue until now to try and update the app to test it.
I myself will do it now, unsure if it'll work or not. Take these release notes with a grain of salt as they mention it's a "potential" fix.

![immagine](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/111080307/e1c31637-f09b-439c-9ffa-459e7a448148)

**Comment by Denizeri24** [comment]
> Hello everyone, Topaz 3.4.0 came out recently and the release notes mention a potential fix to Intel Arc processing issues. I suggest everyone who's had the issue until now to try and update the app to test it. I myself will do it now, unsure if it'll work or not. Take these release notes with a grain of salt as they mention it's a "potential" fix.
> 
> ![immagine](https://user-images.githubusercontent.com/111080307/261267674-e1c31637-f09b-439c-9ffa-459e7a448148.png)

still not working.

**Comment by gargamel314** [comment]
Agreed - It's still erroring out the same as before.

**Comment by Denizeri24** [comment]
Will this problem be solved after 5 years? Because it has not been solved for 6 months.

**Comment by BelleNottelling** [comment]
> Will this problem be solved after 5 years? Because it has not been solved for 6 months.

If you ask me.. they've pretty clearly shown that their only real priority is ensuring that the latest and most popular games work (probably due to benchmarking / review channels).
System stability and productivity has taken what I'd consider to be a significant hit and to me it made the product much worse than it otherwise could be.

I'm not going to tell you what to do with it, but depending on the model of Arc GPU you have, they are still selling on the used market for a fair bit. I sold me A770 LE 16GB on eBay and someone paid nearly $400 for it after shipping and taxes. Once my fees were paid, I got $300 out of it. $300 on eBay can get you an RX 6700 XT which you'll find to be pretty [competitive to an RTX 3070](https://www.youtube.com/watch?v=Wuqfh4xFvks) as long as you aren't looking to do RT. That will also work with TVEAI and it will be perfectly usable.

Do your research and consider your options. If having this work for you isn't important and you're otherwise happy with the Intel GPU by all means keep it & wait things out. Just.. weigh your options & don't allow yourself to think that you are permanently stuck with that card because you've already bought it. Even without having a bunch of extra money to buy something else, there's still options available to you.

**Comment by Denizeri24** [comment]
Will this problem be solved next year?

**Comment by Karen-Intel** [comment]
It is still a WIP on our end. I'm adding your comments to our internal report FYI

Karen

**Comment by gargamel314** [comment]
Just a heads up, **The latest driver from Intel for the Arc series works with Topaz Video Enhance AI version 3.1.9.** 

I tested it with a video file i've been trying to upscale since I posted this thread 8 months ago - it's a video file at 720p trying to upscale to 1080p using the Proteus model, and has failed consistently since March, but this time, it rendered for 7 hours straight and completed. Usually it fails within the first 10-20 minutes.

I can't get it to work with the latest version of VEAI (4.0.3), but this is a start. I have not tried any versions later than 3.1.9. [Here's a link to the discussion on the Topaz forums.](https://community.topazlabs.com/t/new-intel-arc-driver-update/55526/4)

**Comment by gargamel314** [comment]
Hi there - we are approaching one year on this bug report - my renewal is coming up and at this point, I am not renewing this software because the new versions are totally useless to me. I dare say was one of the biggest influences on why I bought this card in early 2023. Topaz Labs has reportedly dropped its support for Intel Arc for their software. Has this bug been shifted to the back burner? Will it continue to be a problem when the next generation of Arc cards come out?

**Comment by EstebanIntel** [comment]
Hi @gargamel314,

We are currently working with Topaz Labs to resolve this issue. The main issue has now been resolved in an internal build, but other minor issues were discovered. We are working to resolve these other issues and hope to have a fix available in the upcoming weeks.

**Comment by gargamel314** [comment]
Just a heads up:

I thought I might try Topaz VEAI again. I downloaded the most recent version for me, version 5.1.4, and with the most recent Intel driver installed (5762), and, much to my surprise, I upscaled a whole bunch of things successfully. I could not get it to crash or fail even once. I even tried the 720p video that crashed Topaz every single time one of you said Topaz was fixed, and it ran for about 4 hours and upscaled the whole thing beautifully. I can't believe it! It works! I haven't tried the latest version of Topaz VEAI, because I did not renew my subscription, and it has expired, so 5.1.4 from about a month ago is the most recent version available to me.

**Comment by Karen-Intel** [comment]
Ty for checking it out @gargamel314! That's great news
As you're the post submitter, are you OK with closing this thread with the behavior you're seeing with Topaz v5.1.4 and driver 5672?
Or do you want to wait until you or other user is able to test with Topaz VEAI?

Please let me know

Karen

**Comment by gargamel314** [comment]
I would recommend waiting until someone else from this thread has tested it with similar results.

**Comment by AuspiciousAlmonds** [comment]
Testing on Topaz v5.1.4 driver 31.0.101.5592 on an ARC A750.

Upscaling models almost all work. Nyx outputs a video of the same length but it's either pitch black completely or mostly black. Everything else seems to work fine

**Comment by Karen-Intel** [comment]
Hey @AuspiciousAlmonds ty for your sharing
Could you please test with v.5672? That's the version Gargamel314 mentioned in his last comment.
We need to confirm the app doesn't crash after a black screen, which is the issue we were pursuing here.
If the app works fine and the issue reported in this thread has been solved, but the output is having issues, you can open a new thread for that specific behavior.

Let us know!

Karen

**Comment by AuspiciousAlmonds** [comment]
Updated the driver and ran an upscale. The screen doesn't go black at any point. No crashes. It's stable

**Comment by Karen-Intel** [comment]
Yeei @AuspiciousAlmonds Ty for confirming that the app doesn't crash or go black anymore with driver v.5672
I think we're ready to close this one out.
@Arturo-Intel could you do the honors please?
Thanks everyone for collaborating all along

Karen
